extensions/tasks/NLUTrainV0/index.ts

```diff
@@ -0,0 +1,38 @@
+import * as tl from 'azure-pipelines-task-lib/task';
+import * as tr from 'azure-pipelines-task-lib/toolrunner';
+import * as utilities from 'nlu-devops-common/utilities';
+
+async function run() {
+    try {
+        const tool = await utilities.getNLUToolRunner();
+        tool.arg('train')
+            .arg('-s')
+            .arg(tl.getInput('service'))
+            .arg('-u')
+            .arg(tl.getInput('utterances'))
+            .arg('-a');
+
+        const serviceSettings = tl.getInput('serviceSettings')
```

Comment:
Once 0.2.5 is published, we'll need to change this to 'modelSettings' and use the '-m' flag below.